database/sql/triggers/insteadof_view_cursos.sql

```diff
@@ -0,0 +1,58 @@
+CREATE OR REPLACE FUNCTION tr_instead_of_view_cursos()
+RETURNS TRIGGER AS $$
+DECLARE
+    -- Declaração do cursor.
+    curso_cursor CURSOR FOR
+        SELECT id, nome, codigo, idioma
+        FROM curso
+        WHERE id = NEW.id;
+
+    -- Variáveis para armazenar os valores originais.
+    old_nome VARCHAR;
```

Comment:
Está com o seguinte erro:

> ERROR:  "old_id" is not a known variable

```suggestion
    old_id INT;
    old_nome VARCHAR;
```